### ISSUE TITLE
Replacing </span> with ] corrupts your entire Twine archive

### ISSUE THREAD

**Issue by pacesmith** [klembot/twinejs#601]
Warning! I had to reinstall Twine to fix this, so beware before you attempt to reproduce.

Use the "Find and replace across the entire story" tool to replace `</span>` with `]`

All nodes of the story become start nodes, you can't edit them, you can't exit the editing window if you attempt to edit one, and attempting to edit one makes all nodes go blank in the overview map.

Then, if you try to load in an archive, it conflicts with the busted one, and you get two stories with the same name, but it thinks there's only one. Deleting my archive and importing an old one did not make the problem go away.

p.s. Hi everyone! New Twine user here.

**Comment by klembot** [comment]
Hi, there should have been a template that appeared when you created this bug that asked whether you were using the desktop, browser version, etc. etc. I need that info from you because right now I tried following the steps you described and it behaved as expected.

**Comment by pace212** [comment]
2.3.3 Windows desktop. If that doesn't help with repro, I'll try it again after IFComp, but I'm too scared to try again right now.

Perhaps you need to have a passage containing a `<span> </span>` ?

**Comment by pace212** [comment]
Reproduced! Just create a start passage consisting of
`<span class="test">[[Reproduce bug 601]]</span>`
and make the "Reproduce bug 601" passage be
`<span class="test">reproduced!</span>`
(I've attached an archive of this in case it's helpful.)

Then use the "Find and replace across the entire story" tool to replace `</span>` with `]`
Be sure to uncheck the "Include passage names" tickybox; that was what I forgot last time, which is probably why neither of us could reproduce it.

Then you ought to be able to reproduce the bug.
